plugin.video.rivedila7/resources/lib/plugin.py

```diff
@@ -4,948 +4,865 @@
-            link_video = re.findall(regex1, html)[0]
+        return
+        # if re.findall(regex1, html):
+        # xbmc.log('REGEX1-----: '+str(re.findall(regex1, html)),xbmc.LOGINFO)
+        #     link_video = re.findall(regex1, html)[0]
     else:
         if re.findall(regex2, html):
-            #xbmc.log('REGEX2-----: '+str(re.findall(regex2, html)),xbmc.LOGINFO)
-            link_video = 'https://awsvodpkg.iltrovatore.it/local/hls/,/content/'+re.findall(regex2, html)[0]+'.mp4.urlset/master.m3u8'
-            #xbmc.log('LINK2-----: '+str(link_video),xbmc.LOGINFO)
+            # xbmc.log('REGEX2-----: '+str(re.findall(regex2, html)),xbmc.LOGINFO)
+            link_video = 'https://awsvodpkg.iltrovatore.it/local/hls/,/content/' + re.findall(regex2, html)[0] + '.mp4.urlset/master.m3u8'
+            # xbmc.log('LINK2-----: '+str(link_video),xbmc.LOGINFO)
         elif re.findall(regex3, html):
-            #xbmc.log('REGEX3-----: '+str(re.findall(regex3, html)),xbmc.LOGINFO)
+            # xbmc.log('REGEX3-----: '+str(re.findall(regex3, html)),xbmc.LOGINFO)
             link_video = re.findall(regex3, html)[0]
         else:
-            #xbmc.log('DECODIFICA DRM',xbmc.LOGINFO)
+            # xbmc.log('DECODIFICA DRM',xbmc.LOGINFO)
             play_dirette(page_video, False)
-            exit()
+            return
         # elif re.findall(regex4, html):
         #     #xbmc.log('REGEX4-----: '+str(re.findall(regex4, html)),xbmc.LOGINFO)
         #     iframe = re.findall(regex4, html)[0]
-        #     req2 = Request(iframe,headers={'user-agent': headers_set['user-agent']})
+        #     req2 = Request(iframe,headers={'user-agent': G.HEADERS_SET['user-agent']})
         #     page2=urlopen(req2)
         #     html2=page2.read()
         #     if re.findall(regex2, html2):
         #         #xbmc.log('REGEX2-B---: '+str(re.findall(regex2, html)),xbmc.LOGINFO)
         #         link_video = str("https:")+re.findall(regex2, html2)[0]
 
-    listitem =xbmcgui.ListItem(titolo_global)
-    listitem.setInfo('video', {'Title': titolo_global})
-    if (thumb_global != ""):
-        listitem.setArt({ 'thumb': thumb_global})
-    listitem.setInfo('video', { 'plot': plot_global })
+    listitem = xbmcgui.ListItem(G.TITOLO, offscreen=True)
+    listitem.setInfo('video', {'Title': G.TITOLO})
+    if G.THUMB != "":
+        listitem.setArt({'thumb': G.THUMB})
+    listitem.setInfo('video', {'plot': G.PLOT})
     if link_video == '':
-        xbmc.log('NO VIDEO LINK',xbmc.LOGINFO)
-        if xbmcgui.Dialog().ok(addon.getAddonInfo('name'), language(32005)):
-            exit()
-    else:
-        listitem.setProperty('inputstream','inputstream.adaptive')
-        listitem.setProperty('inputstream.adaptive.manifest_type','hls')
-        listitem.setPath(link_video)
-        xbmcplugin.setResolvedUrl(handle, True, listitem)
+        xbmc.log('NO VIDEO LINK', xbmc.LOGINFO)
```

Comment:
can you add some sort of plugin identifier to this log line? Anyone checking it on the log won't notice where it's coming from

Reply:
done